exp/services/recoverysigner/cmd/keyset_decrypt.go

```diff
@@ -23,15 +23,15 @@ func (c *KeysetDecryptCommand) Command() *cobra.Command {
 	configOpts := config.ConfigOptions{
 		{
 			Name:        "encryption-kms-key-uri",
-			Usage:       "URI for a remote KMS key used to decrypt the Tink keyset",
+			Usage:       "URI for a remote KMS key used to decrypt the Tink keyset (format: aws-kms://arn:aws:kms:<region>:<account-id>:key/<key-id>)",
 			OptType:     types.String,
 			ConfigKey:   &c.EncryptionKMSKeyURI,
 			FlagDefault: "",
 			Required:    true,
 		},
 		{
 			Name:        "encryption-tink-keyset",
-			Usage:       "Tink keyset to decrypt",
+			Usage:       "Tink keyset to decrypt in JSON format",
```

Comment:
I suggest this change to make it clear it's the decrypted result who is JSON format, and not the Tink keyset.
```suggestion
			Usage:       "Tink keyset to decrypt to JSON format",
```

If the idea was actually to have the Tink keyset in JSON format, I'd suggest:
```suggestion
			Usage:       "Tink keyset in JSON format to be decrypted",
```